protos/grpc/testing/xdsconfig/service.proto

```diff
@@ -0,0 +1,29 @@
+// Copyright 2024 gRPC authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package grpc.testing.xdsconfig;
+
+option go_package = "grpc/interop/grpc_testing/xdsconfig";
+
+import "protos/grpc/testing/xdsconfig/control.proto";
+
+// Service to control the control plane from the test script
+service XdsConfigControlService {
+  // Instructs the server to stop when given resource is requested
+  rpc StopOnRequest(StopOnRequestRequest) returns (StopOnRequestResponse);
```

Comment:
We need to find a better name for this method to avoid the weirdness in the names of the corresponding request and response messages.